ZuluContent/Items/Misc/RaceGate.cs

```diff
@@ -0,0 +1,77 @@
+using System;
+using Scripts.Zulu.Engines.Races;
+using Scripts.Zulu.Utilities;
+using Server.Mobiles;
+
+namespace Server.Items
+{
+    public class RaceGate : Item
+    {
+        public override bool HandlesOnMovement => true;
+        
+        [CommandProperty(AccessLevel.GameMaster)]
+        public ZuluRaceType RaceType { get; set; }
+
+        private bool PickRace(PlayerMobile playerMobile)
+        {
+            if (playerMobile.ZuluRaceType != ZuluRaceType.None)
+            {
+                playerMobile.SendFailureMessage($"You are already a {playerMobile.ZuluRaceType.ToString()}.");
+                return true;
+            }
+            
+            playerMobile.FixedParticles(0x373A, 10, 10, 5007, EffectLayer.Waist);
+                
+            playerMobile.ZuluRaceType = RaceType;
+            playerMobile.Hue = Hue;
+                
+            playerMobile.SendSuccessMessage($"You are now the race of {playerMobile.ZuluRaceType.ToString()}");
+
+            playerMobile.MoveToWorld(new Point3D(1475, 1645, 20), Map.Felucca);
```

Comment:
Probably better to store the coords in `ZuluRaceAttributes`, that way they can have different locations based on race and later we serialize it to JSON so it's not hardcoded.

Reply:
ah yeah good point, done